extensions/2.0/Khronos/KHR_instancing/extensions/2.0/Khronos/KHR_instancing/README.md

```diff
@@ -0,0 +1,64 @@
+# KHR\_instancing
+
+## Khronos 3D Formats Working Group
+
+* Alexey Knyazev
+* Don McCurdy
+* Ed Mackey
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+* Romain Guy
+* Tobias Haeussler
+* John Cooke
+
+## Acknowledgments
+
+* TODO
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+TODO  
+
+## Extending Nodes with per instance attributes
+
+Instancing is defined by adding the `KHR_instancing` extension to any glTF mesh node.  The attributes section contains accessor ids for new TRANSFORM4x3 and ID attribute buffers. 
+For example, the following defines some instancing attributes to a mesh node in the graph. Instancing only applies to mesh nodes.  Applying to nodes rather than meshes allows the 
+same mesh to be attached as normal to nodes while also being instanced.  
+
+```json
+{
+    "nodes": [
+        {
+            "mesh": 0,
+            "name": "teapot",
+            "extensions": {
+                "KHR_instancing": {
+                    "attributes": {
+                        "TRANSFORM4x3": 0,
+                        "ID": 1
+                    },
+                }
```

Comment:
I suspect this is the right design, but just to mention an alternative, the `KHR_instancing` object could also be left empty and the attributes put into the mesh's `primitive.attributes` instead. The per-instance would have different counts, compared to the native mesh attributes, and this would break implementations that don't recognize the extension, if they tried to use those attributes. Recommend keeping as-is.

Reply:
> I chose this design based on what Octane allows; any given mesh can be
referenced by multiple nodes some of which may be instanced...

Agreed, that's another good reason. I'll resolve this suggestion.